instrumentation/cassandra/cassandra-4.4/library/src/main/java/io/opentelemetry/instrumentation/cassandra/v4_4/CassandraAttributesExtractor.java

```diff
@@ -27,6 +27,16 @@ final class CassandraAttributesExtractor
     implements AttributesExtractor<CassandraRequest, ExecutionInfo> {
 
   private static final Logger logger = LoggerFactory.getLogger(CassandraAttributesExtractor.class);
+  private static Field proxyAddressField;
+
+  static {
+    try {
+      proxyAddressField = SniEndPoint.class.getDeclaredField("proxyAddress");
+      proxyAddressField.setAccessible(true);
+    } catch (NoSuchFieldException e) {
+      logger.error("No such a field called \"proxyAddress\"", e);
```

Comment:
I think we typically don't log, e.g. https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/34e0eb67da467be80e5ff67d64c55963ff4015ab/instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/SqsImpl.java#L123